OPERATING.md

```diff
@@ -0,0 +1,151 @@
+# Operating
+
+This document contains requirements and tips for operating the adaptor in a production environment.
+
+# AMQP Message Broker Requirements
+
+* The broker must be configured with a limited number of retries and deadletter queues
+* It is the responsibility of the GP supplier to configure adequate monitoring against the deadletter queues that allows ALL undeliverable messages to be investigated fully.
+* The broker must use persistent queues to avoid loss of data
+* The GP System must persist the relevant message data before acknowledging the message from the queue to avoid loss of data
+
+**Using AmazonMQ**
+
+* A persistent broker (not in-memory) must be used to avoid data loss.
+* A configuration profile that includes settings for [retry and deadlettering](https://activemq.apache.org/message-redelivery-and-dlq-handling.html) must be applied.
+* AmazonMQ uses the scheme `amqp+ssl://` but this **MUST** be changed the to `amqps://` when configuring the adaptor.
+
+**Using Azure Service Bus**
+
+* The ASB must use [MaxDeliveryCount and dead-lettering](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues#exceeding-maxdeliverycount)
+* Azure Service Bus may require some parameters as part of the URL configuration. For example: `LAB_RESULTS_AMQP_BROKERS=amqps://<NAME>.servicebus.windows.net/;SharedAccessKeyName=<KEY NAME>;SharedAccessKey=<KEY VALUE>`
+
+# Mongodb Database Requirements
+
+* The Lab Results Adaptor and Lab Results system communications synchronise through a sequence number mechanism
+* The Mongodb database preserves this synchronisation
+* Deleting the mongodb database and/or its collections will break the link with the Lab Results system
+
+**Amazon Document DB Tips**
+
+In the "Connectivity & security" tab of the cluster a URI is provided to "Connect to this cluster with an application".
+Replace <username>:<insertYourPasswordHere> with the actual mongo username and password to be used by the application.
```

Comment:
`<username>:<insertYourPasswordHere>`
You may need to escape the < and >, or use html entities. This renders just as the colon symbol `:`
```suggestion
Replace \<username\>:\<insertYourPasswordHere\> with the actual mongo username and password to be used by the application.
```

Reply:
Oh yes, thanks for that 👍